libraries/botbuilder-dialogs/src/choices/channel.ts

```diff
@@ -6,28 +6,7 @@
  * Licensed under the MIT License.
  */
 import { TurnContext } from 'botbuilder-core';
-
-/**
- * @private
- */
-export const channels: any = {
-    console: 'console',
-    cortana: 'cortana',
-    directline: 'directline',
-    email: 'email',
-    emulator: 'emulator',
-    facebook: 'facebook',
-    groupme: 'groupme',
-    kik: 'kik',
-    line: 'line',
-    msteams: 'msteams',
-    skype: 'skype',
-    skypeforbusiness: 'skypeforbusiness',
-    slack: 'slack',
-    sms: 'sms',
-    telegram: 'telegram',
-    webchat: 'webchat'
-};
+import { Channels } from 'botframework-schema';
```

Comment:
`import { Channels, TurnContext } from 'botbuilder-core';`

Reply:
@stevengum coolio, deleted schema dependency as instructed, as core already requires and exports it